R/rwebppl.R

```diff
@@ -286,7 +285,7 @@ run_webppl <- function(program_code = NULL, program_file = NULL, data = NULL,
       data_string <- jsonlite::toJSON(data)
       cat(sprintf("module.exports = JSON.parse('%s')", data_string),
           file = file.path(tmp_dir, data_var, "index.js"))
-      add_packages <- c(add_packages, file.path(tmp_dir, data_var))
```

Comment:
the only reason for this was a lifelong habit of not messing with the values of function inputs